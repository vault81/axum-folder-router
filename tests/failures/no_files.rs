use axum_folder_router::folder_router;

#[derive(Clone)]
struct AppState;

#[folder_router("some/non/existing/directory", AppState)]
struct MyFolderRouter();

fn main() {}
