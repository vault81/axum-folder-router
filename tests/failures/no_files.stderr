error: No route.rs files found in the specified directory: 'some/non/existing/directory'. Make sure the path is correct and contains route.rs files.
 --> tests/failures/no_files.rs:6:1
  |
6 | #[folder_router("some/non/existing/directory", AppState)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `folder_router` (in Nightly builds, run with -Z macro-backtrace for more info)
